index.css

```diff
@@ -44,40 +44,167 @@ header .logo {
   color: var(--secondary-color);
 }
 
-header img {
+.top-level img {
   height: 12px;
   align-self: center;
 }
 
 main {
   background: white url(./images/mobile-bg.png) center/contain no-repeat;
-  height: calc(100vh - 43px);
   border-bottom-left-radius: 5rem;
+  height: 626px;
+  margin-bottom: 114px;
+}
+
+.banner {
   width: 100%;
-  padding: 80px 24px 138px 24px;
+  padding: 114px 24px 138px 24px;
   background-color: white;
 }
 
-main .intro {
+.banner .intro {
   font-size: 40px;
-  font-weight: 700;
+  font-weight: var(--bold-font);
 }
 
-main .dev {
+.banner .dev {
   font-weight: var(--regular-font);
   margin-top: 12px;
 }
 
-main h4 {
+.banner h4 {
   font-weight: var(--medium-font);
   text-transform: uppercase;
   margin-top: 12px;
   color: var(--secondary-color);
 }
 
-main ul {
+.banner ul {
   display: flex;
   margin-top: 12px;
   list-style: none;
   gap: 20px;
 }
+
+.grid-container {
+  margin: 0 24px;
+}
```

Comment:
- [ ] Kindly use CSS grid to this projects container in order to align the project cards correctly and professionally and it will help you to maintain the card's responsiveness :100: